src/main/kotlin/mate/academy/Main.kt

```diff
@@ -1,5 +1,12 @@
 package mate.academy
 
+import mate.academy.service.FigureSupplier
+
+private const val ITEMS_QUANTITY = 3
+
 fun main() {
-    // draw all figures here
+    val figureSupplier = FigureSupplier()
+
+    repeat(ITEMS_QUANTITY) { println(figureSupplier.getRandomFigure().getArea()) }
```

Comment:
There is possibility for optimization to avoid code duplication.
Hint 1 -> Please note that repeat lambda contains current position as 'it'. So that you may find out in which half iterator is and choose random or default Figure.

Hint 2 -> if statement returns value, so it is possible to simplify functionality with one println().

Hint 3 -> in this case related to your current code ITEMS_QUANTITY would be 6

Please tag me via chat if you have any questions related to these comments

Reply:
done